src/main/res/layout/progress_recycler_view.xml

```diff
@@ -15,8 +15,15 @@
         android:id="@+id/progress_recycler_view_progress_bar"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_centerVertical="true"
-        android:layout_centerHorizontal="true"
+        android:layout_gravity="center"
         />
 
-</RelativeLayout>
\ No newline at end of file
+    <FrameLayout
+        android:id="@+id/progress_recycler_view_empty_state_container"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
```

Comment:
Cool, I do agree with you. I just didn't realize someone might want it other than at the center, since it was an empty state. Since list is basically empty. So placing in the `center` made sense to me by default :stuck_out_tongue: